ddtrace/contrib/sanic/patch.py

```diff
@@ -99,7 +99,7 @@ def _get_path(request):
         try:
             value = str(value)
         except Exception:
-            # Best effort
+            log.debug("Failed to convert path parameter value to string", exc_info=True)
```

Comment:
With best-effort stuff like the ones in `pytest_bdd` above, it makes sense to me to consistently mention that we're _ignoring_ these errors / failures